src/TTGamesExplorerRebirthUI/Forms/ImageForm.cs

```diff
@@ -66,20 +69,38 @@ private void LoadImages()
         {
             if (_isDDS)
             {
-                _ddsFiles = new DDSImage[_ddsFilesRaw.Count];
+                //_ddsFiles = new DDSImage[_ddsFilesRaw.Count];
 
+                /*
```

Comment:
Same as the other file, if it's commented that can be removed no ?

Reply:
yes, just wanted to keep for refrence or in case of reversal if the code didn't work out well

Reply:
If the code didn't work out well, we can just find the commit and revert the change ;)